snippets/base/admin/actions.py

```diff
@@ -1,8 +1,28 @@
+import csv
+
 from django.db import transaction
+from django.http import HttpResponse
 
 
 @transaction.atomic
 def duplicate_snippets_action(modeladmin, request, queryset):
     for snippet in queryset:
         snippet.duplicate(request.user)
 duplicate_snippets_action.short_description = 'Duplicate selected snippets'  # noqa
+
+
+def export_as_csv(modeladmin, request, queryset):
+    """Adapted from https://books.agiliq.com/projects/django-admin-cookbook/en/latest/export.html"""
+    meta = modeladmin.model._meta
+    field_names = [field.name for field in meta.fields]
+
+    response = HttpResponse(content_type='text/csv')
+    response['Content-Disposition'] = 'attachment; filename={}.csv'.format(meta)
```

Comment:
So this would cause a download of a file called e.g. `base.jobdailyperformance.csv` right? Might be more useful to make it more like `jobdailyperformance.2020-03-23.csv`. Not a requirement, but maybe an improvement for future?

Reply:
Good point, will do!